src/ray/gcs/gcs_server/gcs_actor_scheduler.cc

```diff
@@ -458,6 +458,9 @@ void GcsActorScheduler::CreateActorOnWorker(std::shared_ptr<GcsActor> actor,
   client->PushNormalTask(
       std::move(request),
       [this, actor, worker](Status status, const rpc::PushTaskReply &reply) {
+        // NOTE(rickyx): I couldn't figure out why this is empty reply even though we set
```

Comment:
@rkooo567  any idea on this one?

Reply:
For example, even we do set the push task reply here: https://github.com/ray-project/ray/pull/32726/files#diff-de2456051fd1e5d6c13759cb8cf8bfbe4513b61da7ba69780c1c2ba48a4363d7R190

I verify that this is empty still here. It seems to me we are dropping the reply somewhere or there's another indirection happening.

Reply:
Hmm this is weird. Do you mind investigating a bit more? (or create an issue).

Reply:
So the reason is grpc doesn't return the reply with non-ok status, which I wasn't aware of.